src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -674,7 +677,6 @@ def call(self, hidden_states):
         return pooled_output
 
 
-# Copied from transformers.models.bert.modeling_tf_bert.TFBertSelfOutput
```

Comment:
Why not use the `tf.keras.layers.experimental.EinsumDense` and keep the copy?

Reply:
This one is not possible because the input/output shapes won't be anymore compatible.